client/my-sites/stats/stats-insights/index.jsx

```diff
@@ -23,94 +24,96 @@ import LatestPostSummary from '../post-performance';
 import DomainTip from 'my-sites/domain-tip';
 import Main from 'components/main';
 import StatsFirstView from '../stats-first-view';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { isJetpackSite, getSiteOption } from 'state/sites/selectors';
 
-export default React.createClass( {
-	displayName: 'StatsInsights',
+const StatsInsights = ( props ) => {
+	const { commentsList, followList, gmtOffset, isJetpack, moment, siteId, translate } = props;
+	const moduleStrings = statsStrings();
 
-	propTypes: {
-		commentsList: PropTypes.object.isRequired,
-		followList: PropTypes.object.isRequired,
-		site: React.PropTypes.oneOfType( [
-			React.PropTypes.bool,
-			React.PropTypes.object
-		] ),
-		summaryDate: PropTypes.string,
-	},
-
-	render() {
-		const {
-			commentsList,
-			followList,
-			site,
-		} = this.props;
-
-		const moduleStrings = statsStrings();
-
-		let momentSiteZone = i18n.moment();
-
-		const gmtOffset = get( site.options, 'gmt_offset', null );
-		if ( gmtOffset ) {
-			momentSiteZone = i18n.moment().utcOffset( gmtOffset );
-		}
-
-		const summaryDate = momentSiteZone.format( 'YYYY-MM-DD' );
+	let momentSiteZone = moment();
+	if ( gmtOffset ) {
```

Comment:
Actually, the prior logic was not used here, it was used on the controller to compute the `summaryDate` prop which was not used in the component. The check here is the same check (falsy value) we had in the component.